devel/git/Portfile

```diff
@@ -41,6 +41,7 @@ default_variants    +doc +pcre +credential_osxkeychain +diff_highlight
 
 categories          devel
 license             GPL-2 LGPL-2.1+
+license_noconflict  openssl3
```

Comment:
Setting aside the question of whether git is distributable for the moment, there should never be a `license_noconflict` declaration for a port that isn't listed in the dependencies. It should be used to indicate that one or more of the listed deps are not used in a way that creates a license conflict.

Reply:
Ah, was not aware of this. I'll mark this as draft for now. I'll see if we can address the distributable bit another way.